src/acvm_interop.rs

```diff
@@ -76,3 +78,42 @@ impl ProofSystemCompiler for Gnark {
         gnark_backend::verify_with_vk(circuit, proof, &public_inputs, verification_key).unwrap()
     }
 }
+
+use acvm::acir::circuit::opcodes::BlackBoxFuncCall;
+use acvm::OpcodeResolutionError;
+use acvm::PartialWitnessGenerator;
+use std::collections::BTreeMap;
```

Comment:
Let's gather all the imports at the top of the file following the Rust conventions.